tests/block_tools.py

```diff
@@ -103,7 +103,7 @@
         "DISCRIMINANT_SIZE_BITS": 16,
         "SUB_EPOCH_BLOCKS": 170,
         "WEIGHT_PROOF_THRESHOLD": 2,
-        "WEIGHT_PROOF_RECENT_BLOCKS": 380,
+        "WEIGHT_PROOF_RECENT_BLOCKS": 600,
```

Comment:
Why are we increasing this?

Reply:
we need 3 sub epochs for the sampling since Benedikt asked for little changes, one of them is that last L is the weight of the last 2 sub epochs, if we use 2 like before we have an issue with the sample method since the chain weight would be equal to Last L